composer.json

```diff
@@ -26,7 +26,10 @@
         "symfony/asset": "^3.4|^4.1.1",
         "symfony/config": "^3.4|^4.1.1",
         "symfony/console": "^3.4|^4.1.1",
-        "symfony/contracts": "^1.0",
+        "symfony/cache-contracts": "^1.0",
+        "symfony/translation-contracts": "^1.0",
```

Comment:
We only use contracts coming from this package, could you remove the other ones?